nsone/__init__.py

```diff
@@ -155,7 +155,11 @@ def loadRecord(self, domain, type, zone=None, callback=None,
         import nsone.zones
         if zone is None:
             # extract from record string
-            zone = '.'.join(domain.split('.')[1:])
+            parts = domain.split('.')
+            if len(parts) <= 2:
+                zone = '.'.join(parts)
+            else:
+                zone = '.'.join(domain.split('.')[1:])
```

Comment:
not a big deal, could we reuse `parts` in the else clause? `zone = '.'.join(parts[1:])`